### ISSUE TITLE
compilation question

### ISSUE THREAD

**Issue by olesenm** [adriandt/colm-suite#87]
Nothing urgent, but now that colm, ragel are all in a single repo, figured I'd try to give it a build.

Which branch and which tool should I be using?

_master_ appears to be development activities.

The README states that it is autoconf-based, but after autogen, configure make, it doesn't find the "version.h" file. Tried again with cmake instead (out of source build), but that fails with not finding `<colm/map.h>`.

**Comment by adriandt** [comment]
I broke the build recently. Try a make version.h first. Then make.

I really need to enable nightly building. I've seen people connect that to the github UI. Would be great to figure it out. I have a VM on which it can run.

**Comment by adriandt** [comment]
And yup, master is where I'm doing active development.

**Comment by adriandt** [comment]
Cmake is maintained by @podsvirov I occasionally change the autotools build and he has been updating the cmake side once in a while.

**Comment by podsvirov** [comment]
Hello to all! I will try to restore the CMake project in my free time.

**Comment by adriandt** [comment]
Thank you @podsvirov!

The recent changes are two things:
1. I added another bootstrap pass to allow for transformations that implement parts of colm.
2. I refactored the dependencies (in /colm) so that a single rule generates only one output. I'm currently using tar for this, but would like to switch to some inbuilt feature in colm for packing output files together, then extracting afterwards in the makefile.
